test/distro/diagram-js-minimap.js

```diff
@@ -0,0 +1,26 @@
+const {
+  expect
+} = require('chai');
+
+
+describe('diagram-js-minimap', function() {
+
+  it('should expose CJS bundle', function() {
+
+    const {
+      DiagramJSMinimap
+    } = require('../..');
+
+    expect(DiagramJSMinimap).to.exist;
+  });
+
+
+  it('should expose UMD bundle', function() {
+    const {

```

Comment:
This basically duplicates the test shown above.

UMD means: I can require the thing by the UMD name, not the `main` entry point, and it will return me an instance, usable in CJS.

Reply:
So then we also have a duplicated test in [moddle-xml](https://github.com/bpmn-io/moddle-xml/blob/master/test/distro/moddle-xml.js#L20) 😐

Reply:
Did not recognize until now. :walking_man:

Reply:
Let's ignore this then :rofl:

Reply:
:bee: https://github.com/bpmn-io/moddle-xml/pull/40